src/util.ts

```diff
@@ -23,3 +23,29 @@ export function buildAd4mClient(server: string): Ad4mClient {
 	//@ts-ignore
 	return new Ad4mClient(apolloClient);
 }
+
+export function generateLanguageInitials(name: string) {
+	const split = name.split('-');
+
+	if (split.length === 1) {
+		return name.substring(0, 2);
+	} else {
+		return split[0][0] + split[1][0]
+	}
+}
+
+export function isSystemLanguage(name: string) {
+	return ['languages', 'agent-expression-store', 'neighbourhood-store', 'perspective-language', 'direct-message-language'].includes(name)
+}
+
+export function sanatizeLink(link: LinkExpression) {
```

Comment:
What's sanatize means here? It seems not a correct word.

Reply:
its a misspell